gertie/Package.swift

```diff
@@ -27,7 +27,8 @@ let package = Package(
       dependencies: [
         "Gertie",
         .product(name: "XExpect", package: "x-expect"),
-      ]
+      ],
+      resources: [.process("__fixtures__")]
```

Comment:
can you remove the other (now unnecessary) exclude in `macapp/App/Package.swift`?